Makefile

```diff
@@ -63,7 +63,9 @@ clean: ## Remove build artifacts
 
 .PHONY: switch
 switch: ## Set up the opam switch
+ifeq ($(OPAMSWITCH), )
```

Comment:
LGTM for now. We should have a way to detect if the switch is different from the state of `opam.export` instead.